modules/retail/pages/retail-deploy-terminals-auto.adoc

```diff
@@ -36,19 +37,22 @@ You can enable auto-signing by creating an auto-sign file that contains the key
 
 
 .Procedure: Configuring the Server to Auto-Accept
-. On the {productname} Server, create file [path]``autosign_grains.conf`` and add this line:
+
+. On the {productname} container host, create file [path]``autosign_grains.conf`` and add this line:
 +
 ----
 autosign_grains_dir: /etc/salt/autosign_grains
 ----
++
+// FIXME: also on container host, where dir does not matter?
 . Create a file at [path]``/etc/salt/autosign_grains/autosign_key``, that contains the auto-sign key you specified with {saltboot}:
 +
 ----
 <AUTOSIGN_KEY>
 ----
 +
 For multiple keys, put each one on a new line.
-. Push new files using [systemitem]``mgrclt cp`` command:
+. Push new files into the container using [command]``mgrctl``:
 +
 ----
```

Comment:
I think we can start this procedure like in other places by `mgrctl term` to switch to the container environment and then do all in the container itself.

Reply:
I changed it accordingly.